build/configs/artik055s/audio/defconfig

```diff
@@ -998,7 +1003,9 @@ CONFIG_DEBUG_MM_HEAPINFO=y
 # CONFIG_DEBUG_RTC is not set
 # CONFIG_DEBUG_SPI is not set
 # CONFIG_DEBUG_WATCHDOG is not set
-# CONFIG_DEBUG_TTRACE is not set
+CONFIG_DEBUG_I2S=y
+CONFIG_DEBUG_I2S_ERROR=y
```

Comment:
Are the debug flags required ? In my opinion default configuration should have them turned off.

Reply:
Developers now want to know if any errors in i2s, hence error flag is enabled. It wont spit any normal, verbose or info logs. To enable info logs, CONFIG_DEBUG_I2S_INFO should be enabled.